tests/unit/components/labelPrinting/LabelPrintingForm.spec.js

```diff
@@ -43,7 +43,7 @@ describe('LabelPrintingForm.vue', () => {
 
   describe('methods', () => {
     it('setSuffixOptions ', async () => {
-      expect(labelPrintingForm.suffixOptions.length).toEqual(SuffixList.length)
+      expect(labelPrintingForm.suffixOptions.length - 1).toEqual(SuffixList.length)
```

Comment:
Is this -1 because there is an additional option of 'No suffix'?

Reply:
Might be better to compare the list values rather than just comparing the length as it will also make sure the values are correct